# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/lib/wasm.js
# Commit: 75d829afb4b0
# Full Hash: 75d829afb4b06a3a8b7e91ea15c8a0cf34665fb7
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2025-09-16 12:02 +0300
# Regressor Bug: 1969509
# File Overlap Count: 7
# Overlapping Files: js/src/vm/ArrayBufferObject.cpp, js/src/wasm/WasmMemory.h, js/src/jit-test/tests/wasm/wasm-resizablearraybuffer-shared.js, js/src/jit-test/tests/wasm/memory64/basic.js, js/src/jit-test/tests/wasm/wasm-resizablearraybuffer.js
# Description:
#   Revert "Bug 1969509 - wasm: Update .maxByteLength to report the source wasm max pages in bytes. r=rhunt" for causing javascript engine crashes (bug 1988694)
# ==============================================================================

diff -r 896c8ee9171d -r 75d829afb4b0 js/src/jit-test/lib/wasm.js
--- a/js/src/jit-test/lib/wasm.js	Tue Sep 16 04:43:56 2025 +0300
+++ b/js/src/jit-test/lib/wasm.js	Tue Sep 16 12:02:13 2025 +0300
@@ -519,12 +519,6 @@
 // Max number of memories in a single wasm module.
 let MaxMemories = 100
 
-// Constants related to memory sizes.
-const MaxMemory64PagesValidation = BigInt(Math.pow(2, 37) - 1); // from spec
-const MaxTable64ElemsValidation = 0xFFFF_FFFF_FFFF_FFFFn; // from spec
-const MaxTableElemsRuntime = 10000000; // from WasmConstants.h
-const MaxUint32 = 0xFFFF_FFFF;
-
 // Common array utilities
 
 // iota(n,k) creates an Array of length n with values k..k+n-1